modules/core/keeper/msg_server.go

```diff
@@ -204,32 +204,25 @@ func (k *Keeper) ConnectionOpenConfirm(goCtx context.Context, msg *connectiontyp
 func (k *Keeper) ChannelOpenInit(goCtx context.Context, msg *channeltypes.MsgChannelOpenInit) (*channeltypes.MsgChannelOpenInitResponse, error) {
```

Comment:
This file has the bulk of the changes, they are mostly identical (removing of "LookupModuleByPort" and usage of the modified "Route" method.